tests/tests.c

```diff
@@ -70,29 +70,52 @@ int main(int argc, char** argv) {
 #if defined(MTMALLOC)
```

Comment:
All of this stuff should be moved to tests.h IMO. It's like a page or two of preprocessor before you even see any of the test rig code. It could even be a new header, tests_config.h